### ISSUE TITLE
Share menu: 4 svg alt text issues

### ISSUE THREAD

**Issue by lfrancis** [forumone/peacecorps-site#394]
https://stage-donate.peacecorps.gov/donate/projects-funds/#issue

The four share menu svg elements require alt text remediation for compliance.

Ensures <img> elements have alternate text or a role of none or presentation

```
wcag2a wcag111 section508 section508a 
```

Target: body > .top_image.top_image--custom.wrapper > .share.share--transparent.js-share.drift_corner__tr > .share__box > a:nth-of-type(1) > img
HTML:

<img src="https://stage-donate-peacecorps-static.s3.amazonaws.com/peacecorps/img/i-facebook.svg">

Summary:
Fix any of the following:

```
Element does not have an alt attribute
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible
Element has no title attribute or the title attribute is empty
Element's default semantics were not overridden with role="presentation"
Element's default semantics were not overridden with role="none"
```
